docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -83,13 +83,21 @@ Grafana’s new panel is available only for React-based panels; no Angular-based
 
 ### Organization roles for users logged in through Auth providers
 
-_Available in experimental in all editions of Grafana._
+_Generally available in all editions of Grafana._
 
-We are slowly rolling out a feature toggle for enforcing syncronization of organization roles for organization roles sync from the authentication provider on user sign-in. The feature will prohibit a user from changing organization roles synced with external auth providers.
+We are slowly rolling out a feature toggle for enforcing synchronization of organization roles for organization roles sync from the authentication provider on user sign-in. The feature will prohibit a user from changing organization roles synced with external auth providers.
 
 If you want to enable this feature, you'll need to enable the `onlyExternalOrgRoleSync` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
 
-A reminder that there is a setting to prevent synchronization of organization roles from the authentication provider regardless of their role in the authentication provider, then refer to the `skip_org_role_sync` setting in your Grafana configuration. Refer to [skip org role sync](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync) for more information.
+A reminder that there is a setting to prevent synchronization of organization roles from the authentication provider. Refer to [skip org role sync](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync) for more information.
+
+## Support bundles
+
+_Generally available in all editions of Grafana._
+
+Support bundles provide a simple way to collect information about your Grafana instance through Grafana's user interface. In a few clicks you can create a support bundle containing data about migrations, plugins, settings and more. This is particularly useful when troubleshooting issues with your Grafana instance.
```

Comment:
How is creating a support bundle useful to the user? Do they do something with it, like send it to us so that we can troubleshoot?